workers/unity/Packages/com.improbable.gdk.core/Commands/WorldCommands.cs

```diff
@@ -415,7 +413,7 @@ internal ReceivedResponse(DeleteEntityResponseOp op, Request req, object context
                 {
                     StatusCode = op.StatusCode;
                     Message = op.Message;
-                    EntityId = op.EntityId;
+                    EntityId = new EntityId(op.EntityId);
                     RequestPayload = req;
                     Context = context;
                     RequestId = requestId;
```

Comment:
request ID is still a long here.

Reply:
As the C API uses `uint` in 13 and `long` in 14 (due to a breaking change), I suggest the GDK just sticks to `long` throughout and cast where appropriate now, so that in the far future when 14.0 is released and you move to it, there's less breaking changes to the user.